src/components/BusinessesComponents/BusinessesSelect.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import businessesRequest from '../../api/businesses';
+import Select from '../Select';
+
+export default class BusinessSelect extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+
+        this.state = {
+            businesses: []
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    componentDidMount() {
+       businessesRequest.get().then(businesses => this.setState({ businesses }));
+    }
+
+    handleChange(evt) {
+        console.log(evt);
+    }
+
+    render() {
+        const optionsProps = ({ name, license }) => ({ value: license, key: license, text: name })
+        return (<Select name="businesses"
```

Comment:
for html-ish jsx I want to keep attrs explicit. I find the spread here disturbing.